raiden/transfer/channel.py

```diff
@@ -1671,11 +1674,18 @@ def handle_receive_lock_expired_light(
             payment_id,
             1,
             state_change.lock_expired,
-            True,
+            False,
             LightClientProtocolMessageType.PaymentExpired,
             state_change.lock_expired.recipient
         )
```

Comment:
can you please explain why this changed from `True` to `False`?

also, do you think we could add named parameters here for better readability?

Reply:
Because in that place the message is not signed by the LC yet, the message is being created for the LC.